classes/admin_portal_data.ts

```diff
@@ -0,0 +1,56 @@
+import firebase from '../firebase'
+
+type Marks = {
+  A: number
+  B: number
+  C: number
+  D: number
+}
+// For Review Marks,
+//     A corresponds to Academic Grades (30 pts)
+//     B corresponds to Co-curricular (projects/internships/publications) (30 pts)
+//     C corresponds to Essay Type Question Responses (30 pts)
+//     D corresponds to Extra-curricular (sports/debates/cultural/literary/leadership etc.) (10 pts)
+
+// For Interview Marks,
+//     A corresponds to Technical Questions (30 pts)
+//     B corresponds to Language and clarity of thought (20 pts)
+//     C corresponds to Motivation for higher studies (40 pts)
+//     D corresponds to Overall personality (10 pts)
+
```

Comment:
You can remove these comments for now. We don't want any applicant to see these criterias.